token-allocation/src/config.ts

```diff
@@ -60,6 +60,6 @@ export const makeConfig = (defaultUnit?: string): Config => {
   return {
     priceAdapter,
     defaultCurrency,
-    makeDefaultUnits: makeDefaultUnits(defaultUnit || new Decimal(1e18).toString()),
+    makeDefaultUnits: makeDefaultUnits(defaultUnit || new Decimal(1).toString()),
```

Comment:
Why 1? This should be 0, the default is 0. If I have no units I don't have 1 unit :)

Reply:
The idea here is that you'd give an input of symbols, but won't have to also do a list of units.

E.g. if I give it [BTC, ETH], I shouldn't also have to provide [1, 1] - that should be implied. If I want anything other than 1 unit of each, that should be specified.

Reply:
So for no units, you should just not provide that symbol. I think this makes sense for most applications of this adapter.

Reply:
Aha, I see... Well, it confused me, but I guess it can make sense. I hope this implicit 1 won't turn into an off-by-one error somewhere down the line :)

Please just document it explicitly in the readme, if it's not already.